site/2016/03/23/zf-doctrine-module-release-and-future.rst

```diff
@@ -0,0 +1,65 @@
+
+We are working hard to support all change that Zend Framework done and are
+doing, the community are working very hard to increase the value of each single
+components and we are doing our best to follow this trend.
+
+We decided to do only one post with 4 similar releases, 2 for DoctrineModule, 1
+for DoctrineORMModule and one for DoctrineMongoODMModule.
+
+DoctrineModule 0.10.1
+---------------------
+It's new securty release:
+
+- Drop compatibility with PHP 5.4 `#553 <https://github.com/doctrine/DoctrineModule/pull/553>`_
+- Update doctrine/common ~2.6 `#551 <https://github.com/doctrine/DoctrineModule/pull/551>`_
+
+DoctrineModule 1.0.1
+---------------------
+
+- Drop compatibility with PHP 5.4 `#553 <https://github.com/doctrine/DoctrineModule/pull/553>`_
+- Improve TravisCi build to work with PHP 7 `#553 <https://github.com/doctrine/DoctrineModule/pull/553>`_
+- Update doctrine/common ~2.6 `#551 <https://github.com/doctrine/DoctrineModule/pull/551>`_
+
+DoctrineORMModule 0.10.0
+------------------------
+
+- `#450 <https://github.com/doctrine/DoctrineORMModule/pull/450>`_ Use stable release for DoctrineModule
+- `#443 <https://github.com/doctrine/DoctrineORMModule/pull/443>`_ Added ability to configure the version column in MigrationsConfigurationFactory
+- `#457 <https://github.com/doctrine/DoctrineORMModule/pull/457>`_ Fixed compatibility with Zend\Mvc 2.7
+- `#458 <https://github.com/doctrine/DoctrineORMModule/pull/458>`_ Drop PHP 5.4 and allow PHP 7 on Travis
+
+DoctrineMongoODMModule 0.10.0
+-----------------------------
+
+- `#146 <https://github.com/doctrine/DoctrineMongoODMModule/pull/146>`_ Support for `odm:schema:update` command
+- `#153 <https://github.com/doctrine/DoctrineMongoODMModule/pull/153>`_ Update to stable package versions
+- `#133 <https://github.com/doctrine/DoctrineMongoODMModule/pull/133>`_ Allows null value as DefaultDb
+- `#155 <https://github.com/doctrine/DoctrineMongoODMModule/pull/155>`_ Fix pagination count with eager cursors
+- `#156 <https://github.com/doctrine/DoctrineMongoODMModule/pull/156>`_ Remove support for old PHP versions
+- `#160 <https://github.com/doctrine/DoctrineMongoODMModule/pull/160>`_ Fixed Zend\Mvc 2.7 compatibility
+
+About the future
+----------------
+We start to work on DoctrineModule 2.0, this release will provide support for
+PHP ~5.6 and PHP 7.0, and it will support Zend Framework ~3.0.
+
+*DoctrineModule* 0.x will be update only with security fix, we suggest you to
+update becuase the codebase it's practically the same and we will stop to
+support it from 29/Jannuary/2019.  We are working on DoctrineModule 1.x to
+support ZF 2.6 and 3.0 components, this is one of the main goal of this major
+release.
+
+About DoctrineORMModule and DoctrineMongoODMModule we have in plane for both a
+first stable release when the supports for ZF 2.6 and 3.0 when DoctrineModule
+will be release with ZF 2.6 and 3.
+After that we will start to work on 2.0.0 in order to support DoctrineModule 2.0
```

Comment:
I am not sure to understand that sentence.
Maybe its a bit too far in the future?